README.md

```diff
@@ -396,6 +399,23 @@ subclass will be returned from all SDK methods that previously returned
 an instance of the parent. See `BaseAPIJSONObjectMeta` and `Translator`
 to see how the SDK performs dynamic lookups to determine return types.
 
+# Versions
+We use a modified version of [Semantic Versioning](https://semver.org/) for all changes. See [version strategy](VERSIONS.md) for details which is effective from 30 July 2022.
+
+## Supported Version
+
+Only the current MAJOR version of SDK is supported. New features, functionality, bug fixes, and security updates will only be added to the current MAJOR version.
+
```

Comment:
minor or PATCH release cadence of 2-3 months

Reply:
I'm not sure what do you mean here. can you pls explain?